server/tserver/src/main/java/org/apache/accumulo/tserver/tablet/CompactionRunner.java

```diff
@@ -41,12 +41,18 @@ public void run() {
       return;
     }
 
-    tablet.majorCompact(reason, queued);
+    CompactionStats stats = tablet.majorCompact(reason, queued);
```

Comment:
I would have thought Spotbugs would have dinged this method returning stats that aren't being used...